lib/robots/dor_repo/gis_delivery/load_vector.rb

```diff
@@ -21,31 +21,27 @@ def perform_work
 
           normalizer.with_normalized do |tmpdir|
             schema = Settings.geohydra.postgis.schema || 'druid'
-            # encoding =  # XXX: these are hardcoded encodings for certain druids -- these should be read from the metadata somewhere
-            #   case druid
-            #   when 'bt348dh6363', 'cc936tf6277'
-            #     'LATIN1'
-            #   else
-            #     'UTF-8'
-            #   end
```

Comment:
this TODO comment has been captured in https://github.com/sul-dlss/gis-robot-suite/issues/850 (which i've linked in a comment closer to the use of the encoding value, i.e. by the `run_shp2pgsql` call below)